app/backend/src/couchers/servicers/auth.py

```diff
@@ -97,6 +98,23 @@ def create_session(context, session, user, long_lived, is_api_key=False, duratio
     return token, user_session.expiry
 
 
+def create_api_key(context, session, user):
+    token, expiry = create_session(
+        context, session, user, long_lived=True, is_api_key=True, duration=timedelta(days=365)
+    )
+    send_api_key_email(session, user, token, expiry)
+
+    notify(
+        user_id=user.id,
+        topic="api_key",
+        key="",
+        action="create",
+        icon="wrench",
+        title=f"An admin created an API key for you, please check your email",
+        link=urls.account_settings_link(),
+    )
+
+
```

Comment:
Please move this helper function to `admin.py` and make the `CreateApiKey` method call this function. This is to avoid duplicated code that does approximately the same thing.

Reply:
Did I not already do that? Oops 😬 

Will do!

 - [ ] Move `create_api_key` into `admin.py`
 - [ ] Use `create_api_key` in `CreateApiKey`